railties/lib/rails/test_help.rb

```diff
@@ -16,6 +16,10 @@
 
 require "rails/testing/maintain_test_schema"
 
+if defined?(Capybara)
+  require "rails/testing/capybara"
+end
```

Comment:
Like I mentioned in the PR description, this felt like the most global place to do this extension, since `ActionDispatch::SystemTestCase`, `ActionDispatch::IntegrationTest`, `ActionView::TestCase`, and maybe even [ActionMailer::TestCase](https://github.com/rails/rails/blob/196696e4cc3238a7928f1809c22ebc593816635b/actionmailer/lib/action_mailer/test_case.rb) could benefit from defining `Capybara` extensions.

Having said that, it **feels** like there is a more appropriate place to do so, or an entirely different and more appropriate mechanism.